### ISSUE TITLE
Hide text but show icon?

### ISSUE THREAD

**Issue by xstrex** [rianadon/timer-bar-card#38]
Hi, really enjoy your card, but wondering if there's a way to hide the text (not the time remaining), but show the icon? 

'full_row' would hide both text & icon, be nice if we could hide one or the other, or both. 

```yml
type: custom:timer-bar-card
entity: timer.bathroom_fan
bar_direction: rtl
bar_background: null
text_width: 0px
full_row: 
``` 

Just a suggestion, cheers!

**Comment by rianadon** [comment]
Thank you! I'm glad you're enjoying it!

I should've seen this suggestion coming when I made `full_row`!
However, I'm curious if it's possible to make something similar work with the current set of options.

How about clearing the entity name and increasing `bar_width`?

```yaml
type: custom:timer-bar-card
entity: timer.bathroom_fan
bar_direction: rtl
bar_background: null
text_width: 0px
name:
bar_width: calc(100% - 7em)
```

If this doesn't look right, I'll get around to giving `full_width` an icon-only mode.

**Comment by xstrex** [comment]
I'll give it a shot and report back, thanks again!

**Comment by xstrex** [comment]
So, it's a bit hacky, and doesn't really do what you'd expect. Tried tweaking the values but had some odd results. (added the white background just to visually see what the bar looked like)

<img width="947" alt="Screen Shot 2022-02-23 at 8 35 37 PM" src="https://user-images.githubusercontent.com/12849470/155453268-fbc29e80-0902-45d0-b26a-9a3c816e1dc9.png">

<img width="969" alt="Screen Shot 2022-02-23 at 8 36 22 PM" src="https://user-images.githubusercontent.com/12849470/155453334-24586e53-7ca3-4777-b9f3-2c0d4d163785.png">

While doing this, ran into another quirk I couldn't explain, or get rid of; a non-running timer, while using the filter option creates a blank/empty box; I'd expect the box itself to disappear, since there's no contents, but that's not the case. 
<img width="953" alt="Screen Shot 2022-02-23 at 8 38 51 PM" src="https://user-images.githubusercontent.com/12849470/155453543-106c682a-7a23-4d4c-b85f-e43e94e06eae.png">

**Comment by rianadon** [comment]
Those are quite odd results. I'll push an option for icon-only!

Thank you for the filter bug! That's to be expected right now, but it shall be fixed.

**Comment by rianadon** [comment]
Fixed in v1.16! Please reopen the issue if there are bugs.
